### PR TITLE
[CONFLUENT-KAFKA] Add confluent-kafka

### PR DESCRIPTION
**Pull request by gabrielhartmann**
Service guide needs updating.  Will work on that on subsequent PR.

### CODE REVIEW

**Comment by benclarkwood** [comment]
retest this please

**Comment by benclarkwood** [comment]
@gabrielhartmann The only difference here is that it uses the confluent rather than OSS packages, right?

**Comment by gabrielhartmann** [comment]
@benclarkwood:  There are a few additions to `server.properties.mustache` as well

File: frameworks/confluent/src/main/dist/server.properties.mustache [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+reserved.broker.max.id={{KAFKA_RESERVED_BROKER_MAX_ID}}
+
+unclean.leader.election.enable={{KAFKA_UNCLEAN_LEADER_ELECTION_ENABLE}}
+
+zookeeper.session.timeout.ms={{KAFKA_ZOOKEEPER_SESSION_TIMEOUT_MS}}
+zookeeper.sync.time.ms={{KAFKA_ZOOKEEPER_SYNC_TIME_MS}}
+
+metric.reporters={{KAFKA_METRIC_REPORTERS}}
+confluent.support.metrics.enable={{KAFKA_CONFLUENT_SUPPORT_METRICS_ENABLE}}
```
Comment (benclarkwood) [inline frameworks/confluent/src/main/dist/server.properties.mustache]: found 'em

File: frameworks/confluent/src/test/java/com/mesosphere/sdk/confluent/scheduler/ServiceSpecTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package com.mesosphere.sdk.confluent.scheduler;
+
+import com.mesosphere.sdk.testing.BaseServiceSpecTest;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ServiceSpecTest extends BaseServiceSpecTest {
+    @BeforeClass
+    public static void beforeAll() {
+        ENV_VARS.set("FRAMEWORK_NAME", "kafka");
```
Comment (benclarkwood) [inline frameworks/confluent/src/test/java/com/mesosphere/sdk/confluent/scheduler/ServiceSpecTest.java]: Do we want this to be just `kafka`?

Comment (gabrielhartmann) [inline frameworks/confluent/src/test/java/com/mesosphere/sdk/confluent/scheduler/ServiceSpecTest.java]: probably not, although any string is fine really.

**Comment by gabrielhartmann** [comment]
retest this please

**Pull request closed without merging**